python/lsst/analysis/tools/actions/vector/mathActions.py

```diff
@@ -0,0 +1,132 @@
+
+
+class ConstantValue(VectorAction):
+    """Return a constant scalar value."""
+
+    value = Field[float](doc="A single constant value", optional=False)
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        return ()
+
+    def __call__(self, data: KeyedData, **kwargs) -> Vector:
+        return np.array([self.value])
+
+
+class AddVector(VectorAction):
+    """Calculate (A+B)."""
+
+    actionA = ConfigurableActionField[VectorAction](doc="Action which supplies vector A")
+    actionB = ConfigurableActionField[VectorAction](doc="Action which supplies vector B")
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        yield from self.actionA.getInputSchema()  # type: ignore
+        yield from self.actionB.getInputSchema()  # type: ignore
+
+    def __call__(self, data: KeyedData, **kwargs) -> Vector:
+        vecA = self.actionA(data, **kwargs)  # type: ignore
+        vecB = self.actionB(data, **kwargs)  # type: ignore
+        return vecA + vecB
+
+
+class SubtractVector(VectorAction):
+    """Calculate (A-B)."""
+
+    actionA = ConfigurableActionField[VectorAction](doc="Action which supplies vector A")
+    actionB = ConfigurableActionField[VectorAction](doc="Action which supplies vector B")
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        yield from self.actionA.getInputSchema()  # type: ignore
+        yield from self.actionB.getInputSchema()  # type: ignore
+
+    def __call__(self, data: KeyedData, **kwargs) -> Vector:
+        vecA = self.actionA(data, **kwargs)  # type: ignore
+        vecB = self.actionB(data, **kwargs)  # type: ignore
+        return vecA - vecB
+
+
+class MultiplyVector(VectorAction):
+    """Calculate (A*B)"""
+
+    actionA = ConfigurableActionField[VectorAction](doc="Action which supplies vector A")
+    actionB = ConfigurableActionField[VectorAction](doc="Action which supplies vector B")
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        yield from self.actionA.getInputSchema()  # type: ignore
+        yield from self.actionB.getInputSchema()  # type: ignore
+
+    def __call__(self, data: KeyedData, **kwargs) -> Vector:
+        vecA = self.actionA(data, **kwargs)  # type: ignore
+        vecB = self.actionB(data, **kwargs)  # type: ignore
+        return vecA * vecB
+
+
+class DivideVector(VectorAction):
+    """Calculate (A/B)"""
+
+    actionA = ConfigurableActionField[VectorAction](doc="Action which supplies vector A")
+    actionB = ConfigurableActionField[VectorAction](doc="Action which supplies vector B")
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        yield from self.actionA.getInputSchema()  # type: ignore
+        yield from self.actionB.getInputSchema()  # type: ignore
+
+    def __call__(self, data: KeyedData, **kwargs) -> Vector:
+        vecA = self.actionA(data, **kwargs)  # type: ignore
+        vecB = self.actionB(data, **kwargs)  # type: ignore
+        return vecA / vecB
+
+
+class FractionalDifference(VectorAction):
```

Comment:
Most vector actions in the `mathActions` file end with the suffix `Vector` (all except here and `ConstantValue`). Is it worth considering making that consistent here too?